doc/symbol.md

```diff
@@ -45,4 +45,17 @@ static Napi::Symbol Napi::Symbol::WellKnown(napi_env env, const std::string& nam
 Returns a `Napi::Symbol` representing a well-known `Symbol` from the
 `Symbol` registry.
 
-[`Napi::Name`]: ./name.md
+### For
+```cpp
+static Napi::Symbol Napi::Symbol::For(napi_env env, const std::string& description);
+static Napi::Symbol Napi::Symbol::For(napi_env env, const char* description = nullptr);
```

Comment:
Hi @JckXia ,

Do we have a test for this `nullptr` value...? I am not sure what happens when using a nullptr `napi_value` for the `description` here:

```cpp
  auto forSymb =
      symbObject.Get("for").As<Function>().Call(symbObject, {description});
```

and I think a test would be beneficial